subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -454,14 +538,31 @@ This task will have three inputs and one output:
 
 When you’re writing a custom task class, it’s easy to register properties as inputs or outputs via annotations. To demonstrate, here is a skeleton task implementation with some suitable inputs and outputs, along with their annotations:
 
-++++
-<sample id="customTaskClassWithInputOutputAnnotations" dir="userguide/tasks/incrementalBuild/customTaskClass" title="Custom task class">
-                    <sourcefile file="buildSrc/src/main/java/org/example/ProcessTemplates.java" snippet="custom-task-class"/>
-                    <sourcefile file="buildSrc/src/main/java/org/example/TemplateData.java"/>
-                    <output args="processTemplates" ignoreExtraLines="true" ignoreLineOrder="true"/>
-                    <output args="processTemplates" ignoreExtraLines="true" ignoreLineOrder="true" outputFile="customTaskClassWithInputOutputAnnotationsUpToDate.out"/>
-                </sample>
-++++
+===== Example: Custom task class
+
+[source.multi-language-sample,java]
+.buildSrc/src/main/java/org/example/ProcessTemplates.java
+----
+include::{samplesPath}/userguide/tasks/incrementalBuild/customTaskClass/buildSrc/src/main/java/org/example/ProcessTemplates.java[tag=custom-task-class]
+----
+
+[source.multi-language-sample,java]
+.buildSrc/src/main/java/org/example/TemplateData.java
+----
+include::{samplesPath}/userguide/tasks/incrementalBuild/customTaskClass/buildSrc/src/main/java/org/example/TemplateData.java[]
+----
```

Comment:
This doesn't seem to render properly for me.  

It looks like this on page load:
![image](https://user-images.githubusercontent.com/5387972/43107926-34923588-8ead-11e8-8ffb-92f608fe6d21.png)

And then after clicking on the text with the file names:
![image](https://user-images.githubusercontent.com/5387972/43107954-4e98e102-8ead-11e8-9e77-a34fb14513ae.png)

I see the code fine, but they're not labeled and it looks like it's supposed to be tabbed?